exampleSite/content/dev/missing-images/footer.md

```diff
@@ -3,7 +3,7 @@ fragment = "footer"
 date = "2016-09-07"
 weight = 1200
 
-[logo]
+[asset]
   title = "Logo Title"
```

Comment:
Would move out the title from there. It doesn't completely relate to the image, but to the whole column. I think it should be moved out of the array in a follow on PR.
Aka in general for the footer fragment.

Reply:
Fixed in #309